lib/Command/UpdateAccount.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ * @author Maadix
+ *
+ * Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Mail\Command;
+
+use OCA\Mail\Db\MailAccount;
+use OCA\Mail\Db\MailAccountMapper;
+use OCP\Security\ICrypto;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateAccount extends Command {
+	public const ARGUMENT_USER_ID = 'user-id';
+        public const ARGUMENT_NAME = 'name';
+        public const ARGUMENT_EMAIL = 'email';
+        public const ARGUMENT_IMAP_HOST = 'imap-host';
+        public const ARGUMENT_IMAP_PORT = 'imap-port';
+        public const ARGUMENT_IMAP_SSL_MODE = 'imap-ssl-mode';
+        public const ARGUMENT_IMAP_USER = 'imap-user';
+        public const ARGUMENT_IMAP_PASSWORD = 'imap-password';
+        public const ARGUMENT_SMTP_HOST = 'smtp-host';
+        public const ARGUMENT_SMTP_PORT = 'smtp-port';
+        public const ARGUMENT_SMTP_SSL_MODE = 'smtp-ssl-mode';
+        public const ARGUMENT_SMTP_USER = 'smtp-user';
+        public const ARGUMENT_SMTP_PASSWORD = 'smtp-password';
```

Comment:
Indentation.

Please run the linter with `composer run cs:fix` on your branch

Reply:
I have run the command before pushing. I Hope it is ok now. My vim config was not matching your style at all :-[
Sorry